python/tests/test_account.py

```diff
@@ -1572,14 +1572,19 @@ def test_import_export_online_all(self, acfactory, tmpdir, data, lp):
         original_image_path = data.get_path("d.png")
         chat1.send_image(original_image_path)
 
+        path = tmpdir.join("attachment.txt")
```

Comment:
Add a comment that this ensures import progress is smooth enough?

By the way export progress is currently based on file count rather than file sizes. Maybe makes sense to change this, or is reading file sizes before start of export expensive?

Reply:
> By the way export progress is currently based on file count rather than file sizes. Maybe makes sense to change this, or is reading file sizes before start of export expensive?

I don't know, never tried this out. Although it doesn't seem to be that important to me from a user perspective (esp. if there are tens/hundreds of files, like usual in production).

What _is_ a bit annoying is that the progress is stuck quite some time during the beginning of the export because the large SQL file is exported. Maybe we should just export the sql file at the end instead of at the beginning.